test/instructions_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class DummyClass
+  # This is the class referenced in the test data.
```

Comment:
It would be considerably friendlier to later readers/maintainers if you explain the _why_ here, more than the _what_.

A blank class could leave people scratching their heads, as the test files all imply there should be an `id` method in it. If I'm understanding this correctly, the reason you don't need it here is because you're not actually testing the testing yet — only the instructions file. Assuming so, then it would be a good idea to extend this comment to explain that.